include/ua_plugin_securitypolicy.h

```diff
@@ -386,6 +382,14 @@ UA_SecurityPolicy_getRemoteAsymEncryptionBufferLengthOverhead(const UA_SecurityP
                                                               const void *channelContext,
                                                               size_t maxEncryptionLength);
 
+/* Gets the a pointer to the context of a security policy supported by the
+ * server matched by the security policy uri.
+ *
+ * @param server the server context.
+ * @param securityPolicyUri the security policy to get the context of. */
+UA_SecurityPolicy *
+UA_SecurityPolicy_getSecurityPolicyByUri(const UA_Server *server,
```

Comment:
Can we move this to server_internal.h or similar?
We don't need this in the public API. So lets keep the surface small.

Reply:
This interface (or something similar) is needed by the accesscontrol plugin when adding support for encrypted passwords. I kept it now, but can remove it if you think some other solution would be better.